tools/pg_inspector/readme.md

```diff
@@ -0,0 +1,63 @@
+pg_inspector readme
+===================
+
+pg_inspector is a tool to inspect ManageIQ process caused deadlock or long time blocking in PostgreSQL. pg_inpector inspects database in four steps. You can run `pg_inspector.rb -h` to see all steps, which are displayed in `Operations` section. For each step, you can see its specific options by `pg_inspector.rb operation -h`.
```

Comment:
"pg_inspector is a tool to inspect database locks originating from ManageIQ processes. pg_inpector does this in four steps.  Run `pg_inspector.rb -h` to see each of the steps."

Reply:
by running `pg_inspector.rb operation -h`